src/lib/txhandler.ts

```diff
@@ -187,6 +193,14 @@ export class SologenicTxHandler extends EventEmitter {
 
       if (typeof sologenicOptions.signingMechanism !== 'undefined')
         this.signingMechanism = sologenicOptions.signingMechanism;
+      
+      /**
+       * Initialize client type
+       */
+
+      if (typeof clientOptions.clientType !== 'undefined') {
+        this.clientType = clientOptions.clientType;
+      } 
```

Comment:
in the proposed scenario you can just use:
 if (typeof clientOptions.clientType !== 'undefined') {
        this.clientType = sologenicOptions.clientType;
      }